news/72.breaking

```diff
@@ -0,0 +1,3 @@
+Drop support for Python 3.4 and 3.5.
```

Comment:
I understand that dropping python 2 support and being only python 3+ compatible is a breaking change, but if every time we drop an intermediate python 3+ version we have to make a new major version of each package...

Given the [one year cadence of python 3](https://www.python.org/dev/peps/pep-0602/) releases from now on, dropping almost a python version every year, and thus a major release of each package, just because, sounds way too much to me.

Reply:
If plone.testing is usable outside of Plone, then calling it a major change is good. 
For most of our packages there will usually only be a breaking change if we drop support for a Plone version.